default.hbs

```diff
@@ -41,16 +41,7 @@
                 {{!-- All content gets inserted here, index.hbs, post.hbs, etc --}}
             </main>
 
-            <footer class="gh-foot">
-                <div class="gh-container">
-                    <div class="gh-foot-menu">
-                        {{navigation type="secondary"}}
-                    </div>
-                    <div class="gh-foot-meta">
-                        Published with <a href="https://ghost.org" target="_blank" rel="noopener">Ghost</a>
-                    </div>
-                </div>
-            </footer>
+            {{>footer}}
```

Comment:
Could this be changed to `{{> footer }}`?

Reply:
sure! fixed in new commit